src/main/java/core/basesyntax/Node.java

```diff
@@ -0,0 +1,61 @@
+package core.basesyntax;
+
+public class Node<K, V> {
+    private K key;
+    private V value;
+    private Node<K, V> next;
+
+    public Node(K key, V value, Node<K, V> next) {
+        this.key = key;
+        this.value = value;
+        this.next = next;
+    }
+
+    public Node(K key, V value) {
+        this.key = key;
+        this.value = value;
+    }
+
+    public K getKey() {
+        return key;
+    }
+
+    public void setKey(K key) {
+        this.key = key;
+    }
+
+    public V getValue() {
+        return value;
+    }
+
+    public void setValue(V value) {
+        this.value = value;
+    }
+
+    public Node<K, V> getNext() {
+        return next;
+    }
+
+    public void setNext(Node<K, V> next) {
+        this.next = next;
+    }
```

Comment:
You don't need getters and setters in this nested class course you use them only in your class, there is no additional logic in your setters so I propose you delete them, but the decision is on you ^_^. The same situation will be with your logic in all methods in the class. You just can call and set variables directly without setters and getters.